bftengine/src/bftengine/Reconfiguration.cpp

```diff
@@ -33,7 +33,8 @@ bool ReconfigurationHandler::handle(const WedgeCommand& cmd,
                                     uint32_t,
                                     const std::optional<bftEngine::Timestamp>&,
                                     concord::messages::ReconfigurationResponse&) {
-  LOG_INFO(getLogger(), "Wedge command instructs replica to stop at sequence number " << bft_seq_num);
+  LOG_INFO(getLogger(),
+           "Wedge command instructs replica to stop at next checkpoint after sequence number " << bft_seq_num);
```

Comment:
nit: its the next next checkpoint (for example, if bft_seq_num is 10, then the wedpoint is 300)

Reply:
done